tests/e2e/flowtest/test_cephfilesystem_creation.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.helpers.helpers import create_resource, validate_cephfilesystem
+from ocs_ci.ocs.exceptions import ResourceAlreadyExists
+from ocs_ci.utility import templating
+
+
+logger = logging.getLogger(__name__)
+cephFS_data = templating.load_yaml(constants.CEPHFILESYSTEM_YAML)
```

Comment:
Move these inside test function, dont keep it global

Reply:
I have added it in global as I am using this config in both the test and the fixture function.

Reply:
ack